analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -486,7 +486,12 @@ class MavenSupport(workspaceReader: WorkspaceReader) {
             }
         }
 
-        log.warn { "Unable to find '$artifact' in any of ${remoteRepositories.map { it.url }}." }
+        when (artifact.classifier) {
+            "sources" -> Level.INFO
+            else -> Level.WARN
+        }.also { level ->
```

Comment:
Just a matter of taste probably, but here I'd personally use a variable, and maybe also prefer `if` over `when`:

    val level = if (artifact.classifier == "sources") Level.INFO else Level.WARN

Reply:
done